fontbe/src/features.rs

```diff
@@ -72,12 +76,157 @@ impl Display for NotSupportedError {
     }
 }
 
+// TODO: ask Colin if we need to be threadsafe
```

Comment:
I think if rust lets you compile the code then it's fine. I'm not doing anything sneaky on my side. fea-rs itself is single-threaded.

Reply:
Ah good point, I forgot how good the compiler is when writing that :)